pkgs/build-support/go/module.nix

```diff
@@ -300,8 +300,11 @@ in
       # We do not set trimpath for tests, in case they reference test assets
       export GOFLAGS=''${GOFLAGS//-trimpath/}
 
+      declare -a flags
+      flags+=(''${tags:+-tags=$(concatStringsSep "," tags)})
```

Comment:
Is this not already done in buildGoDir itself?

Reply:
> Is this not already done in buildGoDir itself?

That seems to not have any effect. 

> Is the idea to disable tags for tests specifically?

Nope, they should just be caried over.